src/main/java/io/github/bloepiloepi/pvp/entities/PvpPlayer.java

```diff
@@ -0,0 +1,13 @@
+package io.github.bloepiloepi.pvp.entities;
+
+public interface PvpPlayer {
+    default double getJumpVelocity() {
```

Comment:
There is no reason for this to be here, since it's only used in `CustomPlayer`. Can you move it back?